.travis.yml

```diff
@@ -1,33 +1,41 @@
 language: ruby
+
 rvm:
   - 2.4.1
   - 2.3.0
   - 2.2.3
+
 env:
   global:
     - DB=postgresql
     - RAILS_ENV=test
     - CI_NODE_TOTAL=1
-    - PGPORT=5432
     - PGHOST=localhost
+    - PGPORT=5432
   matrix:
     - CI_NODE_INDEX=0
+
 before_install:
   - gem install bundler -v 1.14.6
-services:
-  - postgresql
+
 addons:
   apt:
-    sources:
-    - precise-pgdg-9.5
     packages:
-    - postgresql-9.5
-    - postgresql-contrib-9.5
     - libcurl3
     - libcurl3-gnutls
     - libcurl4-openssl-dev
-  postgresql: 9.5
+
 bundler_args: --jobs=1 --retry=3
+
+before_install:
+  - "curl -H 'Cache-Control: no-cache' https://raw.githubusercontent.com/fossas/fossa-cli/master/install.sh | sudo bash"
```

Comment:
Why download fossa here?

Reply:
Because... type-o. This got ported over from the Dalton .travis.yml by accident. Removing it now.